### ISSUE TITLE
Support Managing Return Content (Prefer: )

### ISSUE THREAD

**Issue by jjosef** [Asymmetrik/node-fhir-server-core#165]
### Do you want to request a feature, report a bug, or improve documentation?

This is a new feature, based on https://www.hl7.org/fhir/http.html#return

Would like to be able to specify:
```
Prefer: return=minimal
Prefer: return=representation
Prefer: return=OperationOutcome
```
In request headers in order to determine what to return on `create` and `update` returns.

**Comment by Robert-W** [comment]
Hey @jjosef, thanks for submitting this and the associated PR !!. I'll let the team know and we will try to take a look at it as soon as possible. We are a little preoccupied with some internal deadlines but we'll try to get you some feedback on it as soon as we can.

**Comment by Robert-W** [comment]
PR merged and deployed as version 1.4.2. If you have any issues, please feel free to open a new issue.
